### PR TITLE
Feature/TRACTION-69, TRACTION-71, TRACTION-72

### PR DESCRIPTION
**Pull request by rishabh-trac**
# Critical Changes

# Changes
Created custom labels with translation supported for Hindi, Spanish, French languages
Updated the community and custom lightning and web components to include labels for multilingual support

# Issues Closed
TRACTION-69, 
TRACTION-71, 
TRACTION-72

### CODE REVIEW

File: force-app/main/default/lwc/contactEditForm/contactEditForm.js [hunk 1]
```diff
@@ -12,6 +12,35 @@ import deleteCredential from '@salesforce/apex/ContactEditFormController.deleteC
 import updateCredential from '@salesforce/apex/ContactEditFormController.updateCredential';
 import searchAccountLookup from '@salesforce/apex/StaffAvailabilityController.searchAccountLookup';
 import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import STAFF_INFORMATION from '@salesforce/label/c.Staff_Information';
```
Comment (cwico) [inline force-app/main/default/lwc/contactEditForm/contactEditForm.js:15]: is this namespace safe?

File: force-app/main/default/lwc/contactEditForm/contactEditForm.js [hunk 2]
```diff
@@ -38,6 +67,37 @@ export default class ContactEditForm extends LightningElement {
 		lookupErrors: [],
 		notifyViaAlerts: false // Use alerts instead of toast to notify user
 	};
+	labels = {
```
Comment (cwico) [inline force-app/main/default/lwc/contactEditForm/contactEditForm.js:70]: Super cool

File: unpackaged/config/community/experiences/Traction_Thrive1/views/dashboard.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
       "regions" : [ {
         "components" : [ {
           "componentAttributes" : { },
-          "componentName" : "%%%NAMESPACE_OR_C%%%:overviewDashboard",
+          "componentName" : "c:overviewDashboard",
```
Comment (cwico) [inline unpackaged/config/community/experiences/Traction_Thrive1/views/dashboard.json]: need %%%NAMESPACE_OR_C%%% for the cci script namespace injection

File: unpackaged/config/community/experiences/Traction_Thrive1/views/home.json [hunk 4]
```diff
@@ -18,7 +18,7 @@
         "showAvatar" : "Desktop and Mobile",
         "showContactInfo" : "Desktop and Mobile"
       },
-      "componentName" : "%%%NAMESPACE_OR_C%%%:contactCard",
+      "componentName" : "c:contactCard",
```
Comment (cwico) [inline unpackaged/config/community/experiences/Traction_Thrive1/views/home.json]: need %%%NAMESPACE_OR_C%%% for the cci script namespace injection

File: unpackaged/config/community/experiences/Traction_Thrive1/views/patients.json [hunk 5]
```diff
@@ -26,7 +26,7 @@
       "regions" : [ {
         "components" : [ {
           "componentAttributes" : { },
-          "componentName" : "%%%NAMESPACE_OR_C%%%:patientTracker",
+          "componentName" : "c:patientTracker",
```
Comment (cwico) [inline unpackaged/config/community/experiences/Traction_Thrive1/views/patients.json]: need %%%NAMESPACE_OR_C%%% for the cci script namespace injection

File: unpackaged/config/community/experiences/Traction_Thrive1/views/supplies.json [hunk 6]
```diff
@@ -26,7 +26,7 @@
       "regions" : [ {
         "components" : [ {
           "componentAttributes" : { },
-          "componentName" : "%%%NAMESPACE_OR_C%%%:allResources",
+          "componentName" : "c:allResources",
```
Comment (cwico) [inline unpackaged/config/community/experiences/Traction_Thrive1/views/supplies.json]: need %%%NAMESPACE_OR_C%%% for the cci script namespace injection

**Review by cwico — CHANGES_REQUESTED** [review]
Thank you for doing this. I have some namespace injection issues in this PR for you to look at and and update. 

I know there are some hardcoded ids. Hope it works fine in the package. SFDC is weird.

File: force-app/main/default/lwc/contactEditForm/contactEditForm.js (hunk 1 above)
Comment (rishabh-trac) [reply force-app/main/default/lwc/contactEditForm/contactEditForm.js:15]: yup it is

File: unpackaged/config/community/experiences/Traction_Thrive1/views/customHome.json [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    "regionName" : "featured",
+    "type" : "region"
+  }, {
+    "components" : [ {
+      "componentAttributes" : {
+        "showAccountInfo" : "Desktop and Mobile",
+        "showAvatar" : "Desktop and Mobile",
+        "showContactInfo" : "Desktop and Mobile"
+      },
+      "componentName" : "%%%NAMESPACE_OR_C%%%:contactCard",
```
Comment (cwico) [inline unpackaged/config/community/experiences/Traction_Thrive1/views/customHome.json:21]: Yay!

**Review by cwico — APPROVED** [review]

**Pull request merged**